deploy/chart/templates/30_02-clusterserviceversion.crd.yaml

```diff
@@ -189,21 +189,28 @@ spec:
                   items:
                     type: object
                     required:
-                    - name
+                    - group
```

Comment:
Most of the e2e failures are regarding API validation for `ClusterServiceVersions` on tests you didn't modify. You may need to make some fields optional to fix them.